book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- Hey, let's add the :rentals to the attr_accessor as this helps in establishing the many-to-many relationship between the book and a person as the rentals is the intercessor between them. 


  `attr_accessor :title, :author, :rentals`